docs/specifications/secure-launch-specification.rst

```diff
@@ -0,0 +1,887 @@
+
```

Comment:
This file is a newer revision of `/assets/secure-launch-specification.pdf` and probably should be replaced in `/specifications/Secure_Launch.md` but it is a different filetype, so I am unsure.

Reply:
Skip it for now, we have (will have) dedicated task for updating the specification later.